admission-webhook/webhook.go

```diff
@@ -296,6 +332,11 @@ func (webhook *webhook) mutateCreateRequest(pod *corev1.Pod) (*admissionv1beta1.
 					"value": contents,
 				})
 			}
+		} else if contentsAnnotationPresent {
```

Comment:
This check is already performed in the validation function above. Since this is pure validation (rather than any kind of mutation), can we remove the check here?

Reply:
Fair enough :)